test/specs/errors/conf/error_schema/error_schema.json

```diff
@@ -47,8 +47,114 @@
           "name": "Valid Table Name"
         }
       }
+    },
+    "main_delete_table": {
```

Comment:
Rename this. It's confusing, you're not testing delete feature here. Name it `main` or something.